pkg/edgeview/src/basics.go

```diff
@@ -0,0 +1,451 @@
+		case "mdns":
+			helpOn("mdns[/intf-name][/service]", "display zeroconfig related information")
+			helpExample("mdns/eth0", "display mDNS for default service 'workstation' on interface 'eth0'", true)
+			helpExample("mdns/bn1/https", "display mDNS for service 'https' on bridge 'bn1'", false)
+			helpExample("mdns", "display mDNS for default service 'workstation' on all UP interafces", false)
+		case "nslookup":
+			helpOn("nslookup[/<ip or name>]", "display domain name and dns server information")
+			helpExample("nslookup/www.amazon.com", "display DNS information on www.amazon.com", true)
+			helpExample("nslookup/8.8.8.8", "display DNS information on address 8.8.8.8", false)
+		case "ping":
+			helpOn("ping[/<ip or name>]", "ping to 8.8.8.8 from all the UP interfaces or ping a specific address")
+			helpExample("ping", "ping to 8.8.8.8 from each source IP address of the intefaces", true)
+			helpExample("ping/192.168.1.1", "ping the address of 192.168.1.1", false)
+		case "route":
+			helpOn("route", "display all the ip rule and their ip table entries")
+		case "socket":
+			helpOn("socket", "display all the ipv4 litening socket ports and established ports")
+		case "speed":
+			helpOn("speed[/intf-name]", "run speed test and report the download and upload speed")
+			helpExample("speed/wlan0", "run speed test on interface wlan0", true)
+		case "tcp":
+			helpOn("tcp/ip-address:port[/ip-address:port...][/proxy[@ip-addr]]", "tcp connection to the ip addresses for services, local mapping ports 9001 and above")
+			helpExample("tcp/192.168.1.1:8080", "points your browser to the locally listening port and http browsing 192.168.1.1:8080", true)
+			helpExample("tcp/10.1.0.2:80/10.1.0.2:8081", "points your browser to the locally listening ports and http browsing remote 10.1.0.2 both 80 and 8081 ports", false)
+			helpExample("tcp/proxy/localhost:5903", "https proxy to locally listening ports and vnc viewer to #3 port on device", false)
+			helpExample("tcp/proxy@10.1.2.3", "https proxy and specify the address of DNS name server for URL lookup", false)
+		case "tcpdump":
+			helpOn("tcpdump/intf-name/[options]", "tcpdump on the interface, can specify duration with -time, default is 60 sec")
+			helpExample("tcpdump/eth0/", "run tcpdump on eth0 with default 60 seconds or maximum of 100 entries", true)
+			helpExample("'tcpdump/eth0/port 443' -time 10", "run tcpdump on eth0 and port 443 with 10 seconds", false)
+		case "trace":
+			helpOn("trace[/<ip or name>]", "traceroute to www.google.com and zedcloud server, or to specified ip or name, 10 hops limit")
+			helpExample("trace", "traceroute to www.google.com and to zedcloud server", true)
+			helpExample("trace/www.microsoft.com", "run traceroute to www.microsoft.com", false)
+		case "url":
+			helpOn("url", "display url metrics for zedclient, zedagent, downloader and loguploader")
+		case "wireless":
+			helpOn("wireless", "display the iwconfig wlan0 info and wpa_supplicant.conf content")
```

Comment:
Btw. Input to wwan service (config) is `/run/wwan/config.json` and the output from wwan service are `/run/wwan/status.json` and `/run/wwan/metrics.json`. Might be helpful to include that in the command output as well.

Reply:
sure.